Library/Formula/gif2png.rb

```diff
@@ -0,0 +1,22 @@
+class Gif2png < Formula
+  homepage "http://www.catb.org/~esr/gif2png/"
+  url "http://www.catb.org/~esr/gif2png/gif2png-2.5.10.tar.gz"
+  sha256 "3a593156f335c4ea6be68e37e09994461193f31872362de4b27ef6301492d5fd"
+
+  depends_on "libpng"
+
+  def install
+    ENV.deparallelize # parallel install fails
```

Comment:
The upstream author does not appear to have a bug reporting tool, so I email him instead.